pkg/db/columnhandler.go

```diff
@@ -154,7 +154,7 @@ func (ch *ColumnHandler) ColumnName(field string) (string, error) {
 	return "", fmt.Errorf("entity '%s' has no field '%s': cannot resolve column name", ch.entity, field)
 }
 
-//ColumnNamesCsv returns the CSV string of the column names
+// ColumnNamesCsv returns the CSV string of the column names
```

Comment:
https://github.com/golang/go/wiki/CodeReviewComments#initialisms
```suggestion
// ColumnNamesCSV returns the CSV string of the column names
```

Reply:
Agree on your points. A fix would imply to change the wrong fct-name as well to have it consistant. Please see my comments below, why we try to minimise any changes in the old reconciler (we are close to the transition phase and will drop the old reconciler in next few months).